crates/nu-command/src/filesystem/rm.rs

```diff
@@ -241,6 +243,10 @@ fn rm(
                         Value::Error {
                             error: ShellError::SpannedLabeledError(msg, e.to_string(), span),
                         }
+                    } else if quiet {
+                        Value::Nothing {
+                            span: Span::new(0, 0),
```

Comment:
Instead of `Span::new(0, 0)`, you can use `call.head`. The zero span will just point at random stuff at the beginning of the file memory and using it is discouraged except of tests where it doesn't matter.

Reply:
Changed that now, using `span` as it captures `call.head`